src/components/search/detailed/AppSearchResults.js

```diff
@@ -59,7 +42,7 @@ export default function AppSearchResults(props) {
     const { searchTerm, updateResultCount, baseId } = props;
     const [appsSearchKey, setAppsSearchKey] = useState(APPS_SEARCH_QUERY_KEY);
     const [appsSearchQueryEnabled, setAppsSearchQueryEnabled] = useState(false);
-
+    const [detailsResource, setDetailsResource] = useState(null);
```

Comment:
Looks like this is not used?
There's also a GitHub Actions `warning` on this line.
I guess the `DetailsDrawer` is missing in this file?

_edit_: I just saw your response to an outdated comment that this is fixed in #170 ...